pytorch_lightning/trainer/trainer_conf.py

```diff
@@ -226,15 +226,15 @@ class WandbConf:
     node=ObjectConf(cls="pytorch_lightning.loggers.wandb.WandbLogger", params=WandbConf()),
 )
 
+defaults = [{"logger": "wandb"}, {"trainer": "trainer"}]
+
 
 @dataclass
-class PytorchLightningTopConfig(DictConfig):
+class PLConfig(DictConfig):
+    defaults: List[Any] = field(default_factory=lambda: defaults)
     trainer: Any = MISSING
-    logger: Any = MISSING
-    profiler: Any = MISSING
-    checkpoint: Any = MISSING
-    early_stopping: Any = MISSING
-
-
-cs.store(name="pl_config", node=PytorchLightningTopConfig)
+    logger: Any = None
```

Comment:
What are the proper types of these fields?

Reply:
Well the type should be some base class for the logger configuration. Although none of the classes at the moment have a base class for logger, early stopping, etc for me to use for type checking.

Reply:
If they have anything in common, you can factor it out into a base class and use it here.

Reply:
could be good even as a tagging interface.